app/models/oauth_client.rb

```diff
@@ -29,6 +31,7 @@
 class OAuthClient < ApplicationRecord
   belongs_to :integration, polymorphic: true
   has_many :remote_identities, dependent: :destroy
+  has_many :oauth_client_tokens, dependent: :destroy
```

Comment:
Good catch. DB already has a ON DELETE CASCADE set anyway. =)

Reply:
Yes, it was